macro_utils_h_generator/macro_utils.tt

```diff
@@ -35,55 +35,55 @@
 #define P_OR_NULL(p) (((p)!=NULL)?(p):"NULL")
 #define WP_OR_NULL(p) (((p)!=NULL)?(p):L"NULL")
 
-#define TOSTRING_(x) #x
-#define TOSTRING(x) TOSTRING_(x)
+#define MU_TOSTRING_(x) #x
+#define MU_TOSTRING(x) MU_TOSTRING_(x)
 
-#define IFCOMMA(N) C2(IFCOMMA_, N)
-#define IFCOMMA_0
-#define IFCOMMA_2
+#define MU_IFCOMMA(N) MU_C2(MU_IFCOMMA_, N)
+#define MU_IFCOMMA_0
+#define MU_IFCOMMA_2
 <#for(var i=4;i<=nMacroParameters;i+=2)#>
 <#{#>
-#define IFCOMMA_<#=i#> ,
+#define MU_IFCOMMA_<#=i#> ,
 <#}#>
 
-#define IFCOMMALOGIC(N) C2(IFCOMMALOGIC_, N)
+#define IFCOMMALOGIC(N) MU_C2(IFCOMMALOGIC_, N)
```

Comment:
>IFCOMMALOGIC [](start = 8, length = 12)

It looks like the intended prefix for the renamed MACROs is "MU_". This macro does not seem to have the "MU_"

Reply:
Addressed.